### DISCUSSION TITLE
How do I know which formatter is being invoked?

### DISCUSSION THREAD

**Discussion by PookieBuns** [neoclide/coc.nvim#4735] [Q&A]
When I call 
`:call CocActionAsync('format')`
and open the log, it only shows
`2023-08-29T10:43:23.639 INFO (pid:5072) [attach] - receive notification: format []`
Is there any way to quickly see which formatter is being invoked when multiple extensions are active for the file type?

Currently, I have to go into each extension's log output to see if it received a format call to check. 
Is there a better way to do this?

**Comment by chemzqm** [comment]
Use command `:CocCommand prettier.openOutput` to checkout the output from coc-prettier.
